### PR TITLE
added empty notification hub and interface with one un-implemented me…

### PR DESCRIPTION
**Pull request by mattmarnien**
…thod, added empty hubtest class

### CODE REVIEW

**Comment by escalonn** [comment]
@mendac017 you can use those labels on non-US PRs too if you want. but whenever this code gets PRed into the US branch, it would need to be in a tdd-fail PR again. i don't see a lot of benefit to it - i suggest either having PRs like this go directly to `us-notifications`, or if you want things broken up more than that, make a case for dividing notifications into multiple user stories.

now that i look at the diff here actually, there's changes that i htink don't quite fit in a tdd-fail PR, so the label doesn't make too much sense.

**Comment by ghost** [comment]
@escalonn Matt has moved the target branch to us-notifications and the merge branch to notificationHub.

**Comment by mattmarnien** [comment]
This includes:
- a skeleton Notifications Hub (4 unimplemented methods)
- a skeleton INotificationHub Interface with signatures for 4 methods
- A test project with tests for the 4 methods, currently failing due to unimplemented status

File: FakebookNotifications/FakebookNotifications.WebApi/Hubs/NotificationHub.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+using FakebookNotifications.WebApi.HubInterfaces;
+
+namespace FakebookNotifications.WebApi.Hubs
+{
+    public class NotificationHub : Hub<INotificationHub>
+    {
+
+        public new IHubCallerClients  Clients { get; set; }
+
+        //Send a notfication to all clients
```
Comment (ghost) [inline FakebookNotifications/FakebookNotifications.WebApi/Hubs/NotificationHub.cs]: notfication should be notification

Comment (ghost) [reply FakebookNotifications/FakebookNotifications.WebApi/Hubs/NotificationHub.cs]: on all of these comments

File: FakebookNotifications/FakebookNotifications.Testing/NotificationHubTest.cs [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            await hub.SendCaller("caller", "test");
+
+            // assert
+            // checks to see if a message was sent to the caller-user, once, and not other users
+            mockClients.Verify(c => c.Caller, Times.Once);
+            mockClients.Verify(o => o.AllExcept(caller), Times.Never);
+        }
+
+        /// <summary>
+        /// Tests notifcation hub method to send a notification to a specific user
```
Comment (ghost) [inline FakebookNotifications/FakebookNotifications.Testing/NotificationHubTest.cs]: spelling on this comment

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]
These look good. These tests fail because they are throwing not implemented.

**Review by townerr — APPROVED** [review]
Looks good

**Pull request merged**